README.md

```diff
@@ -24,7 +24,7 @@ since the early 2003.
 
 License:
  * OpenASIP project source code is licensed with LGPL v2.1.
- * Generated hardware description files are licensed with a more liberal MIT license.
+ * Generated hardware description files are public domain.
```

Comment:
Sorry going back and forth, but thinking again, I think we have to use the MIT license since that's what the original authors agreed for the HDLs. But it's permissive so it's fine.